### ISSUE TITLE
Is it expected that v2 trained models underperform compared to v1 trained models on identical settings?

### ISSUE THREAD

**Issue by kalomaze** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#297]
[v1_vs_v2_Examples.zip](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/files/11490137/v1_vs_v2_Examples.zip)

5 minute dataset. I also used the [Mangio fork which adds 'crepe' as a training option](https://github.com/Mangio621/Mangio-RVC-Fork) for both of these models.
Maybe for a model with a bigger dataset like 40 minutes, v2 outperforms?

**Comment by kalomaze** [comment]
I have been informed it is much easier to 'overtrain' v2 models. Both are 1000 epochs. Will look into that first, but I'm running into an issue where TensorBoard cuts out at 6k steps, when the full model in theory has 20+ thousand steps, due to a batch size of 4 with a dataset of about ~5 minutes (so roughly ~100 post-processing wavs)

**Comment by RVC-Boss** [comment]
Try both 20epoch model? Also you can try the latest v2 model pretrained weights.
Now it's an experimental model and therefore performence may be decrease in some cases. We won't publish the release until it's tested completely.

**Comment by kalomaze** [comment]
The "v2" files here *are* using the experimental v2 weights. The "v1" example is the normally trained weights model.

According to TensorBoard, the model is 6,000 steps. This may be innaccurate and it is improperly reading the logs? (1000 epochs, 5 minute dataset, batch size 4). I am told that, if /g/total starts rising again on the graph after peaking, it is a sign the model is overtraining. If that is correct, 300-400 epochs was the correct ending point, and 1000 epochs is far too much (for the v2 model only). I can see this being correct for such a small dataset. This person also told me that they saw this overtraining effect start at around 1000 epochs on a 40 minute dataset (again, just on v2).

<img width="458" alt="image" src="https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/66376113/186e9b9c-781b-4452-9e04-7f2edd02d6a1">

With this in mind, the 350 epochs model seems to beat the 1000 epochs v1 model.

[v1_1000_EPOCHS_VS_v2_350_EPOCHS.zip](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/files/11494247/v1_1000_EPOCHS_VS_v2_350_EPOCHS.zip)

**Comment by RVC-Boss** [comment]
The 350e_v2 version is significantly better than 1000e_v2 (1st floor) version.

**Comment by RVC-Boss** [comment]
In the earliest version, the max total_epoch is 100, than updated to 200. Somebody says more epoch may be better, needs 1000epoch, so now max total_epoch is 1000. But I think less epoch training is the advantage of RVC.
